gatsby/content/blog/2023/01/2023-01-31-synapse-1.76.mdx

```diff
@@ -0,0 +1,85 @@
+- Releases
+author: H. Shay
+---
+
+Greetings matrix fans! We've published Synapse [version 1.76](https://github.com/matrix-org/synapse/releases/tag/v1.76.0)
+as the new stable release this week. Synapse admins are encouraged to upgrade
+to it at their convenience. Please take a look at the [upgrade notes](https://matrix-org.github.io/synapse/v1.76/upgrade.html#upgrading-to-v1760)
+for any important information about upgrading.
+
+## Announcements
+
+The big news is that faster joins for all has arrived! When joining a room for the first time,
+Synapse 1.76.0 will request a partial join from the other server by default. Previously, server admins
+had to opt-in to this using an experimental config flag.
+
+Server admins can opt out of this feature for the time being by setting
+
+```
+experimental:
+    faster_joins: false
+```
+
+in their server config.
+
+For everyone else, after a faster join Synapse considers that room "partially joined". In this state, you should be able to
+
+- read incoming messages;
+- see incoming state changes, e.g. room topic changes; and
+- send messages, if the room is unencrypted.
+
+Synapse has to spend more effort to complete the join in the background. Once this finishes, you will be able to
+
+- send messages, if the room is encrypted;
+- retrieve room history from before your join, if permitted by the room settings; and
+- access the full list of room members.
+
+Practically, this means that the experience of joining a large room over federation should be greatly improved. In past
+versions of Synapse, joining a large room such as Matrix HQ over federation could take up to 30 mins. This time has been
+cut to ~20-30 seconds in the new release. Those of you who have been watching this know this has been a huge undertaking,
+big shout out to the team for getting it over the line! We're excited the community to try it, as always if you encounter
+any problems you are encouraged to file an [issue](https://github.com/matrix-org/synapse/issues/new?assignees=&labels=&template=BUG_REPORT.yml).
+
+
+### Changes to the account data replication streams
+
+Please note that Synapse has changed the format of the account data and devices replication streams (between workers).
+This is a forwards- and backwards-incompatible change: v1.75 workers cannot process account data replicated by v1.76
+workers, and vice versa.
+
+Once all workers are upgraded to v1.76 (or downgraded to v1.75), account data and device replication will resume as normal.
+
+
+## The new stuff
+
+### The default room version has now been updated to [v10](https://spec.matrix.org/v1.5/rooms/v10/)
+
+Room version 10 is now the default room version in Synapse 1.76.0. This room version builds on v9 to introduce a new
+`knock_restricted` join rule, allowing prospective members to more easily join such a room. The `knock_restricted` join
+rule allows for prospective members of a room to join either through knocking or through "join_restrictions", which allows
```

Comment:
```suggestion
rule allows for prospective members of a room to join either through knocking or through `join_restrictions`, which allows
```
either that or lose the underscore — basically either human speak or computer speak, but choose one and use the right style